nest-clamav-scanner/src/minio-client/minio-client.service.ts

```diff
@@ -57,12 +58,20 @@ export class MinioClientService {
   }
 
   //function which checks if file exists in minio bucket
-  public async fileExists(bucketName: string, fileName: string) {
+  public async fileExists(
+    bucketName: string,
+    fileName: string,
+  ): Promise<BucketItemStat> {
     try {
       return await this.minioService.client.statObject(bucketName, fileName);
     } catch (error) {
-      this.logger.error(error);
-      return false;
+      if (error instanceof Error) {
+        this.logger.error(error.message);
+      } else {
+        this.logger.error('fileExists is throwing non Error');
```

Comment:
I those cases, you can still log whatever error is, in this case you lose only information you have

```
this.logger.error(`fileExists is throwing non Error: ${error}`);
```